components/opc-ua/src/Microsoft.Azure.IIoT.OpcUa.Edge.Publisher/src/Extensions/PublishedDataSetEventModelEx.cs

```diff
@@ -52,7 +52,7 @@ public static class PublishedDataSetEventModelEx {
                 AttributeId = null,
                 IndexRange = null,
                 SamplingInterval = TimeSpan.Zero,
-                PendingAlarms = publishedEvent.PendingAlarms?.Clone() ?? null,
+                ConditionHandling = publishedEvent.ConditionHandling.Clone(),
```

Comment:
would it be possible that ConditionHandling is null here?

Reply:
Clone() is an extension method and checks for null, and in that case returns null.